app/src/main/java/com/jjswigut/matters/ui/MatterListFragment.kt

```diff
@@ -0,0 +1,44 @@
+package com.jjswigut.matters.ui
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.jjswigut.matters.ui.MatterListRecyclerViewAdapter
+import com.jjswigut.matters.R
+import com.jjswigut.matters.dummy.DummyContent
+
+/**
+ * A fragment representing a list of Items.
+ */
+class MatterListFragment : Fragment() {
+
+    private var columnCount = 1
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_matter_list, container, false)
```

Comment:
I know it wasn't in our original plan, but if you wanted to switch over to https://developer.android.com/topic/libraries/view-binding?authuser=2 that would be nice!

Reply:
done!

Reply:
awesome! ViewBinding is definitely the way to go over findViewbyId / kotlin synthetics.

Reply:
I understand why it's better than findviewbyid(memory cost searching the whole project everytime) but why is it better than kotlin synthetics?

Reply:
the main benefit that comes to mind is nullability. you can reference a kotlin synthetic in any fragment / activity (most of the time by accident) and there are no compile time guarantees that you aren't making a mistake. so if you have a textview in fragment A, you can reference that textview in fragment B and you'll get a crash at runtime. viewbinding makes that not possible